language-service/src/services/yamlCompletion.ts

```diff
@@ -223,22 +223,40 @@ export class YAMLCompletion {
 		});
 	}
 
+	private checkPropertyArray(properties: string[], property: string): boolean {
+		if (!properties || !properties.length) {
+			return true;
+		}
+
+		let contained: boolean = false;
+		properties.forEach(listProperty => {
+			if (listProperty == property) {
+				contained = true;
```

Comment:
Nit: could just return true here and return false if we get past the foreach.

Reply:
early returns make code harder to read